readme.md

```diff
@@ -16,114 +15,98 @@ Member States (from the European Economic Area, which encompasses more countries
-
-[LINK to app spec repos]
-
-https://github.com/ehn-dcc-development/ehn-dcc-valuesets
-https://github.com/ehn-dcc-development/ehn-dcc-schema
-https://github.com/ehn-dcc-development/hcert-spec
-
-## Documentation
-
-https://github.com/ehn-dcc-development/hcert-trust [RYAN: deprecated]
-
-## Implementations
-
-[TODO]
-
-## Full Stack
-
-These repositories contain implementations of the entire DCC stack, and are the perfect starting place for anyone who wants to understand how the DCC system works in detail.
-They can also be used during developement to validate your own implementation.
-
-The full stack consists of:
-
-1. Creating the JSON payload conform the schema.
-2. Packing according the HCERT specifications (so sign + pack in COSE/CBOR, zip, base45 then format to HCERT string and optionally rendering as a QR).
-3. Validation by scanning QR (or processing HCERT), checking the signature and optionally checking the JSON payload against the schema.
-
-https://github.com/ehn-dcc-development/ehn-sign-verify-python-trivial
-
-
-## HCERT
-
-[Explain that HCERT implementation contains the entire stack]
-
-[TODO half of these repos contain an almost full stack impelementain]
-
-https://github.com/ehn-dcc-development/hcert-kotlin
-https://github.com/ehn-dcc-development/hcert-app-swift
-https://github.com/ehn-dcc-development/python-hcert
-https://github.com/ehn-dcc-development/hcert-service-kotlin
-https://github.com/ehn-dcc-development/hcert-dotnet
-https://github.com/ehn-dcc-development/hcert-java
-
-## COSE
-
-[Explain]
-
-https://github.com/ehn-dcc-development/cose-js
-https://github.com/ehn-dcc-development/COSE-JAVA
-
-
-## Base45
+## Specifications
 
-[Explain]
+The DCC specifications have been split into four repositiros:
 
-https://github.com/ehn-dcc-development/base45-php
-https://github.com/ehn-dcc-development/base45-cs
-https://github.com/ehn-dcc-development/base45-ansi-C
-https://github.com/ehn-dcc-development/base45-swift
-https://github.com/ehn-dcc-development/base45-js
-https://github.com/ehn-dcc-development/base45-java
+* [hcert-spec](https://github.com/ehn-dcc-development/hcert-spec) specifies the container and encoding formats used by the DCC.
+* [ehn-dcc-valuesets](https://github.com/ehn-dcc-development/ehn-dcc-valuesets) provides the valueset datastructures and a snapshot of the valuesets (the current versions are published on the DCCG Gateway.
+* [ehn-dcc-schema](https://github.com/ehn-dcc-development/ehn-dcc-schema) specifies the DCC schema (the specification of the data stored within the DCC).
+* [dgc-business-rules](https://github.com/ehn-dcc-development/dgc-business-rules) specifies the Business Rules framework.
 
+You can find these repositories pinned on our github [homepage](https://github.com/ehn-dcc-development).
 
-## Verifiers
+We have a team of maintainers who monitor the above mentioned pinned repositories. If you have questions or suggestions please feel free to open an issue on the respective repository.
```

Comment:
The question is whether this team will really exist in a continuous sense after roughly June 1st. Could the EC make an email address that forwards to the relevant people, e.g. Stavros, Konstantin, etc.?

Reply:
we do for the coming time - there will always be people maintaining it as long as it's in maintenance.